lib/jekyll/netlify/generator.rb

```diff
@@ -1,16 +1,16 @@
+require_relative 'environment'
 module Jekyll
   module Netlify
     # Netlify plugin generator
     class Generator < Jekyll::Generator
       safe true
 
       def generate(site)
+        env = Environment.new
         if netlify?
           site.config['netlify'] = load_netlify_env
-          if production?
-            ENV['JEKYLL_ENV'] = 'production'
```

Comment:
Can we also set `ENV['JEKYLL_ENV'] = env.jekyll_env` , in case someone access it that way?